app/ota_client.py

```diff
@@ -1254,9 +1254,9 @@ def set_update_ecuinfo(self, update_info):
         return ecu_found
 
     def update(self, ecu_update_info):
-        return self._update(ecu_update_info, reboot=False)
+        return self._update(ecu_update_info)
 
-    def _update(self, ecu_update_info, reboot=False):
```

Comment:
Could you remove `def update()` and rename `def _update()` to `def update()`. _update is already unnecessary.

Reply:
Sure! It seems that this update -> _update design is just for passing some extra parameters. 
(P.S., This kinds of pattern also can be seen in many other places. )